setup.py

```diff
@@ -50,27 +50,27 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-bigquery >= 0.21.0',
-    'google-cloud-bigtable >= 0.21.0',
-    'google-cloud-happybase >= 0.20.0',
-    'google-cloud-core >= 0.21.0',
-    'google-cloud-datastore >= 0.21.0',
-    'google-cloud-dns >= 0.21.0',
-    'google-cloud-error-reporting >= 0.21.0',
-    'google-cloud-language >= 0.21.0',
-    'google-cloud-logging >= 0.21.0',
-    'google-cloud-monitoring >= 0.21.0',
-    'google-cloud-pubsub >= 0.21.0',
-    'google-cloud-resource-manager >= 0.21.0',
-    'google-cloud-storage >= 0.21.0',
-    'google-cloud-translate >= 0.21.0',
-    'google-cloud-vision >= 0.21.0',
-    'google-cloud-runtimeconfig >= 0.21.0',
+    'google-cloud-bigquery >= 0.21.0, < 0.22dev',
+    'google-cloud-bigtable >= 0.21.0, < 0.22dev',
+    'google-cloud-happybase >= 0.21.0, < 0.22dev',
```

Comment:
Note: I **just** released this https://pypi.python.org/pypi/google-cloud-happybase/0.21.0